File: easy/6_is_leap_year/is_leap_year_test.go
package main

import "testing"

func TestIsLeapYear(t *testing.T) {
	if !isLeapYear(2000) {
		t.Error("expected true, got false")
	}
	if isLeapYear(2001) {
		t.Error("expected false, got true")
	}
	if isLeapYear(2002) {
		t.Error("expected false, got true")
	}
	if isLeapYear(2003) {
		t.Error("expected false, got true")
	}
	if !isLeapYear(2004) {
		t.Error("expected true, got false")
	}
	if isLeapYear(2100) {
		t.Error("expected false, got true")
	}
}

File: easy/6_is_leap_year/is_leap_year_test.py
import unittest
from is_leap_year import *

class TestIsLeapYear(unittest.TestCase):
    def testLeapYear(self):
        self.assertEqual(is_leap_year(2000), True)
        self.assertEqual(is_leap_year(2001), False)
        self.assertEqual(is_leap_year(2002), False)
        self.assertEqual(is_leap_year(2003), False)
        self.assertEqual(is_leap_year(2004), True)
        self.assertEqual(is_leap_year(2100), False)